packages/ui/src/styles/components/iab-consent-dialog.module.css

```
@@ -444,6 +444,19 @@
 		height: 0.75rem;
 	}
 
+	/* Text for assistive technology only, kept out of the visual layout. */
+	.srOnly {
+		position: absolute;
+		width: 1px;
+		height: 1px;
+		padding: 0;
+		margin: -1px;
+		overflow: hidden;
+		clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)